data/accountManager.go

```diff
@@ -79,10 +79,10 @@ func (manager *AccountManager) Keys(rnd basics.Round) (out []account.Participati
 // StateProofKeys returns a list of Participation accounts, and their stateproof secrets
 func (manager *AccountManager) StateProofKeys(rnd basics.Round) (out []account.StateProofSecretsForRound) {
 	for _, part := range manager.registry.GetAll() {
-		if part.OverlapsInterval(rnd, rnd) {
+		if part.StateProof != nil && part.OverlapsInterval(rnd, rnd) {
 			partRndSecrets, err := manager.registry.GetStateProofSecretsForRound(part.ParticipationID, rnd)
 			if err != nil {
-				manager.log.Errorf("error while loading round secrets from participation registry: %w", err)
```

Comment:
why removed the err wrapping?

Reply:
It is due to the fact that we don't return a wrapped error here, just log the message. 
Current, the log will contain a line like this:
```
error while loading round secrets from participation registry: %!w(*fmt.wrapError=&{...})
```

Reply:
ah, right, logging, thanks!